releasenotes/notes/45800.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: traffic-management
+issue:
+- 45798
+releaseNotes:
+- |
+  **Fixed** an issue where Istiod might crash when a cluster is deleted.
```

Comment:
document this can happen when xds cache disabled

Reply:
Thanks, fixed